packages/@aws-cdk/aws-stepfunctions/lib/states/distributed-map/distributed-map-resource.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Distributed Map State Resources
+ */
+export enum DistributedMapResource {
```

Comment:
I don't think this provides a good user experience. What are these values for? What do they mean? 
I could only find `getObject` in the docs you've linked.

Preferably we would refactor this to represent the semantic meaning instead of leaking the technical implementation. At least we should provide documentation explaining each of these.